src/app/index.js

```diff
@@ -1,43 +1,49 @@
-var PushClient = require('./PushClient');
-var CSVParser = require('./CSVParser');
-var request = require('request');
-const zlib = require('zlib');
-var path = require('path');
-var fs = require('fs')
+/* might as well use 'const' for all requires */
+const PushClient = require('./PushClient'),
+  CSVParser = require('./CSVParser'),
+  request = require('request'),
+  zlib = require('zlib');
 
-
-// TOKEN TO PUSH TO THE COVEO PUSH API
-var pushtoken = process.env.COVEO_PUSH_TOKEN || 'YOUR_KEY';
+// We call them 'API Keys' in the Admin UI, we should use the same terminology.
+var pushApiKey = process.env.COVEO_PUSH_TOKEN || 'YOUR_KEY';
 // ID OF THE COVEO ORGANIZATION TO PUSH TO
 var organizationId = process.env.COVEO_ORGANIZATIONID || 'ORG';
 // ID OF THE SOURCE TO PUSH TO
 var sourceId = process.env.COVEO_SOURCEID || 'SOURCEID';
 
 
 // START THE COVEO PUSH CLIENT
-var myPushClient = new PushClient({
-  organizationId: organizationId, 
-  sourceId: sourceId, 
-  pushtoken: pushtoken
-}, { retry: true, attempts: 5, delay: 3000 });
+let myPushClient = new PushClient({
+    organizationId: organizationId,
+    sourceId: sourceId,
+    pushApiKey: pushApiKey
+  },
+  { retry: true, attempts: 5, delay: 3000 }
+);
 
-CSVParser.parseCSV(path.resolve(process.cwd(), 'src', 'InsuranceDemoData_ER2.csv'), documents => {
+// should not hardcode the input, here's how to get the name from the command line.
+let csvInputFile = process.argv[2];
+CSVParser.parseCSV(csvInputFile, documents => {
   var count = 0;
   documents.forEach((document) => {
-    if (document.Locationofdocument && document.Locationofdocument.match(/\.pdf$/i)) {
+    /*
+    We should use better names in the header line as describe what we expect in the README
+     */
+    if ( (document.Locationofdocument||'').match(/\.pdf\s*$/i) ) {
```

Comment:
Sure, I saw this more as an example than an actual re-usable program 👍 
Might as well make it re-usable.